### PR TITLE
Improve build and execution

### PR DESCRIPTION
**Pull request by mrmm**
Build process improvement:
 - Run the build process **inside a container** 

Dockerized solution improvement :
 - Make image configurable using env vars
 - Shrink Image size from **251.2 MB**  to **12.17 MB**

Metrics:
 - Add response time metric

Release:
 - Added bin directory and SHA512 sum for the binary 

Documentation
 - Detailled metrics explanation
 - How to configure the image 
 - How to build the images

### CODE REVIEW

**Comment by hnlq715** [comment]
Thanks for your contribution! I'll review this pr ASAP

File: nginx_vts_exporter.go [hunk 1]
```diff
@@ -340,16 +350,17 @@ func fetchHTTP(uri string, timeout time.Duration) func() (io.ReadCloser, error)
 }
 
 var (
-	listenAddress   = flag.String("telemetry.address", ":9113", "Address on which to expose metrics.")
+	listenAddress   = flag.String("telemetry.address", ":9913", "Address on which to expose metrics.")
```
Comment (hnlq715) [inline nginx_vts_exporter.go]: May we should keep compatible with the former listen port 9113？I'm wondering what's the purpose to change it?

File: nginx_vts_exporter.go [hunk 2]
```diff
@@ -144,7 +150,7 @@ type Exporter struct {
 	serverMetrics, upstreamMetrics, cacheMetrics map[string]*prometheus.GaugeVec
 }
 
-func newServerMetric(metricName string, docString string, labels []string) *prometheus.GaugeVec {
+func newServerMetric(metricName string, docString string, labels []string, namespace string) *prometheus.GaugeVec {
```
Comment (hnlq715) [inline nginx_vts_exporter.go]: the namespace here could also be a global variable

Comment (mrmm) [inline nginx_vts_exporter.go]: This solves the problem when you want to run the vts-nginx-exporter on different Nginx instances and you have only one Prometheus scrapper.

File: nginx_vts_exporter.go (hunk 1 above)
Comment (mrmm) [inline nginx_vts_exporter.go]: It was just a random port number, it is configurable with **METRICS_ADDR** Env variable if you run it inside a container.

File: nginx_vts_exporter.go (hunk 2 above)
Comment (hnlq715) [inline nginx_vts_exporter.go]: Yes, I totally agree. But the point here is we do not need pass the namespace as args, just use the global variable `namespace` instead, this should avoid changing too many codes. What do you think? :)

File: nginx_vts_exporter.go (hunk 1 above)
Comment (hnlq715) [inline nginx_vts_exporter.go]: Yes, this was a random port number, and it's configurable. But it's better for us to keep the former default port number 9113, isn't it?

File: nginx_vts_exporter.go (hunk 2 above)
Comment (mrmm) [inline nginx_vts_exporter.go]: Well we can set the global variable `namespace` from Env variable so we can use it anywhere without changing any function header.

**Pull request merged**